earth_enterprise/src/fusion/autoingest/SConscript

```diff
@@ -149,6 +149,8 @@ for plugin in plugins:
     env.Command('plugins/'+plugin+'Asset.cpp',
                 [ 'AssetGenPublicCpp.pl', 'AssetGen.pm', src ],
                 [ "${SOURCES[0]} ${SOURCES[2]} $TARGET" ])
+    env.Depends('plugins/'+plugin+'Asset.cpp', 'plugins/'+plugin+'Asset.h')
```

Comment:
I don't think the .cpp needs to depend on the .h. More likely the .os needs to depend on both the .cpp and .h. Same comment in the other file.

Reply:
yep that makes sense